tests/auth_tests/test_checks.py

```diff
@@ -47,8 +47,9 @@ class CustomUserBadRequiredFields(AbstractBaseUser):
         errors = checks.run_checks(self.apps.get_app_configs())
         self.assertEqual(errors, [
             checks.Error(
-                "The field named as the 'USERNAME_FIELD' for a custom user model "
-                "must not be included in 'REQUIRED_FIELDS'.",
+                "The 'USERNAME_FIELD' in your custom user model "
+                "is currently set to '%s'. Please remove '%s' "
+                "from your 'REQUIRED_FIELDS' definition." % (CustomUserBadRequiredFields.USERNAME_FIELD, CustomUserBadRequiredFields.USERNAME_FIELD),
```

Comment:
When adding the hint please put the actual expected string, rather than interpolating.